app/models/recipe.rb

```diff
@@ -1,5 +1,8 @@
 require 'net/http'
 class Recipe < ActiveRecord::Base
+
+    has_and_belongs_to_many :users
```

Comment:
Every user has multiple recipes, and every recipe can be associated with multiple users. This is a good, sensible design. Hopefully adding the concept of a "favorited" recipe will go off of this smoothly